src/index.js

```diff
@@ -85,6 +85,15 @@ module.exports = (opts = {}) => {
 			if (resultPluginIndex === -1) {
 				throw new Error("Plugin missing from options.");
 			}
+			// resolve and fileResolve can't be used together
```

Comment:
Thank you for thinking about the backwards compatibility. Anyway, I'd prefer to introduce the breaking change and bump the major version. So, let's remove the original `resolve` version, replace it with yours, and update the readme. Feel free to use the `resolve` option name for this.